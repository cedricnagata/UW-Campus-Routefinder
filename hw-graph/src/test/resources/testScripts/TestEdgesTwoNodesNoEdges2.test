# Create a graph
CreateGraph Graph

# Adds two nodes to the graph
AddNode Graph n1
AddNode Graph n2

# List the children of n2 in the graph
ListChildren Graph n2