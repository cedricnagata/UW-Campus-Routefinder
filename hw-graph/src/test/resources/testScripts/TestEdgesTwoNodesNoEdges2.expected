# Create a graph
created graph Graph

# Adds two nodes to the graph
added node n1 to Graph
added node n2 to Graph

# List the children of n2 in the graph
the children of n2 in Graph are: